core/src/main/java/org/springframework/security/authorization/event/AuthorizationEvent.java

```diff
@@ -110,4 +112,14 @@ public AuthorizationResult getAuthorizationResult() {
 		return this.result;
 	}
 
+	/**
+	 * Get {@link ResolvableType} of this class.
+	 * @return {@link ResolvableType}
+	 * @since 6.5
+	 */
+	@Override
+	public ResolvableType getResolvableType() {
```

Comment:
Since this class doesn't have generics, I think it will have unintended consequences if placed here. Will you please move this down to `AuthorizationGranted` and `AuthorizationDeniedEvent`?